types/pretty/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for pretty 2.0
+// Project: https://github.com/jonschlinkert/pretty
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+export interface PrettyOptions {
+    ocd: boolean;
+}
+
+export function pretty(str: string, options?: PrettyOptions): string;
```

Comment:
It looks like the underlying library exports a function.  Should this be `function pretty... ;  export = pretty;`?

Reply:
@amcasey Yeah, I'm still picking up the best way to define type defs here, your right. Fixed